### PR TITLE
[#8761] dist3 blacklist

### PR DESCRIPTION
**Pull request by hawkowl**
https://twistedmatrix.com/trac/ticket/8761

### CODE REVIEW

File: twisted/python/_dist3.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
+# -*- test-case-name: twisted.python.test.test_dist3 -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Support for installing Twisted on Python 3.
+
+Only necessary while parts of Twisted are unported.
+
+@var modules: A list of modules that have been ported,
```
Comment (glyph) [inline twisted/python/_dist3.py]: This variable doesn't exist any more.

File: twisted/python/_dist3.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    e.g. "twisted.python.versions"; a package name (e.g. "twisted.python")
+    indicates the corresponding __init__.py file has been ported
+    (e.g. "twisted/python/__init__.py"). To reduce merge conflicts, add new
+    lines in alphabetical sort.
+
+@var testModules: A list of test modules that have been ported, e.g
```
Comment (glyph) [inline twisted/python/_dist3.py]: This variable doesn't exist any more.

File: twisted/python/_dist3.py [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    "twisted.python.test.test_versions". To reduce merge conflicts, add new
+    lines in alphabetical sort.
+
+@var testDataFiles: A list of filenames that are data files used by tests.
```
Comment (glyph) [inline twisted/python/_dist3.py]: This variable doesn't exist any more.

File: twisted/python/_dist3.py [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    These are generally used by another Python process that the test case
+    spawns.
+
+@var almostModules: A list of any other modules which are needed by any of the
```
Comment (glyph) [inline twisted/python/_dist3.py]: This variable doesn't exist any more.

File: twisted/python/_dist3.py [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+    modules in the other two lists, but which themselves have not actually
+    been properly ported to Python 3.  These modules might work well enough to
+    satisfy some of the requirements of the modules that depend on them, but
+    cannot be considered generally usable otherwise.
+
+@var modulesToInstall: A list of all modules that should be installed on
```
Comment (glyph) [inline twisted/python/_dist3.py]: This variable doesn't exist any more.

File: twisted/python/_dist3.py [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+    Python 3.
+"""
+
+from __future__ import absolute_import, division
+
+notPortedModules = [
```
Comment (glyph) [inline twisted/python/_dist3.py]: This variable isn't documented.

**Comment by rodrigc** [comment]
Looks OK, merge it.

File: src/twisted/python/_dist3.py [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+@var notPortedModules: A list of fully-qualified Python names that correspond
+    to modules in Twisted are not ported, for exclusion during installation.
+"""
+
+from __future__ import absolute_import, division
+
+notPortedModules = [
+    "twisted.conch.tap",
+    "twisted.conch.ttymodes",
+    "twisted.internet._ssl",
```
Comment (alex) [inline src/twisted/python/_dist3.py]: This file no longer exists and should be removed.

Comment (glyph) [inline src/twisted/python/_dist3.py]: More importantly - there should probably be some checking for this condition; if any of the files don't exist a test should probably fail.

Comment (hawkowl) [inline src/twisted/python/_dist3.py]: There is: https://github.com/twisted/twisted/pull/501/files/9535a4069979e35226c6a3a329bd154f1c586604#diff-8df27befe87e3feb795ca53f050639a5L29 -- we removed this file a few hours ago and I haven't merged forward. :)

Comment (glyph) [inline src/twisted/python/_dist3.py]: Hooray :).  In that case it wouldn't be able to land if this wasn't fixed anyway; awesome.

**Pull request merged**